java/src/client/facade/Chat.java

```diff
@@ -0,0 +1,23 @@
+package client.facade;
+
+/**
+ * Created by audakel on 9/16/16.
+ */
+public class Chat extends Facade{
```

Comment:
True - but as I believe my code was submitted as a pr before the pr which changed facades to shared, there was no shared facades folder. Normally you accept prs in chronological order